src/hooks/common/useWebViewMessage.ts

```diff
@@ -0,0 +1,13 @@
+import { useUserAgent } from '~/hooks/common/useUserAgent';
+
+export function useWebViewMessage() {
+  const { isAndroid, isIos, isMobile } = useUserAgent();
+
+  const postMessage = (type: string, data?: any) => {
+    if (!isAndroid() && !isIos() && !isMobile()) return;
```

Comment:
```suggestion
    if (!isMobile()) return;
```

`isMobile`이 `isAndroid() || isIos()`의 구현체라 이렇게 작성해도 될 거 같아요!

Reply:
피드백 감사합니다!! 반영했어요~